src/settings/MatchProfiles/MatchProfiles.js

```diff
@@ -265,7 +265,7 @@ export class MatchProfiles extends Component {
                 resultsLabel={label}
                 defaultSort="name"
                 actionMenu={this.renderActionMenu}
-                resultsFormatter={resultsFormatter(searchTerm, selectRecord, selectedRecords)}
+                resultsFormatter={listTemplate('matchProfiles', searchTerm, selectRecord, selectedRecords)}
```

Comment:
Please, create an enum for entity keys ('matchProfiles', 'jobProfiles' and others)